code/modules/client/preferences_savefile.dm

```diff
@@ -80,6 +80,12 @@
 		WRITE_FILE(S["max_chat_length"], max_chat_length)
 		WRITE_FILE(S["see_chat_non_mob"], see_chat_non_mob)
 
+	if(savefile_version == 43)
+		var/datum/loadout_manager/manager = load_loadout_manager()
+		if(istype(manager))
+			loadout_manager.loadouts_data = convert_loadouts_list(manager?.loadouts_data)
```

Comment:
Why is this not on loadouts through the own versioning it has

Reply:
until this PR, the loadout_manager datum itself was saved via jatum. 
Now, we avoid jatum and are saving only the the list. Hence we are no long saving the loadout manager version, because we are no longer saving/loading the manager

If the savefile is older than 44, we have to load the manager one more time and extract the list from it

Reply:
ok